packages/app/src/contact-support/reducers/contact-support.reducer.test.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Let's, also write tests for other cases

Reply:
https://github.com/mudita/mudita-center/pull/561/commits/1372585d5696ef8c14a7e57d166d94b1a11eb76b